.env.example

```diff
@@ -11,3 +11,9 @@ MAILER_API_KEY=theTokenOfTheMailerService
 MAILER_EMAIL=theEmailUsedByTheMailerAsSender
 API_BASE_URL=theApiBaseUrlToInsertInTheLinksSharedInEmails
 TOKEN_KEY=TheTokenKey
+
+
```

Comment:
@Ramadhan0 I'm confused by the format of these env variables. Why is there a space in front of them? Please, use the format of the other variables in this file and ensure the key names are descriptive.